playground/src/main/java/com/ramcosta/samples/playground/commons/DestinationsAppExtensions.kt

```diff
@@ -33,6 +33,8 @@ fun DirectionDestination.DrawerContent(
         GoToProfileConfirmationDestination,
         SettingsScreenDestination,
         ThemeSettingsDestination -> Unit
```

Comment:
```suggestion
        ThemeSettingsDestination,
        ProfileSettingsScreenDestination -> Unit
```

Reply:
Yeah go with this then. Maybe this is indeed something I messed up while merging the last time 😁
Since it’s only on the sample app, it’s totally possible I didn’t ran it 🙂

Reply:
Sorry for the confusion!

Reply:
No worries!